corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore.py

```diff
@@ -131,6 +131,11 @@ def testOverwriteCache(self):
         self.assertIsInstance(restore_config_cached.get_payload(), CachedResponse)
         self.assertNotIsInstance(restore_config_overwrite.get_payload(), CachedResponse)
 
+        # even cached responses change the sync log id so they are not the same
```

Comment:
It is an edge case which probably doesn't happen that often (though I don't have data to support that assumption). If we include the device ID in the key then I think we can go ahead.